packages/core/src/actions/smartAccount/sendRawUserOperation.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  ChainNotFoundError,
+  type Chain,
+  type Client,
+  type Hash,
+  type Transport,
+} from "viem";
+import type { SmartContractAccount } from "../../account/smartContractAccount";
+import { isBaseSmartAccountClient } from "../../client/isSmartAccountClient.js";
+import { AccountNotFoundError } from "../../errors/account.js";
+import { IncompatibleClientError } from "../../errors/client.js";
+import { InvalidUserOperationError } from "../../errors/useroperation.js";
+import { isValidRequest } from "../../utils/index.js";
+import type { SendRawUserOperationParameters } from "./types";
+
+export const sendRawUserOperation: <
```

Comment:
why redefine this? it's already being added on the smartAccountClient: https://github.com/alchemyplatform/aa-sdk/blob/main/packages/core/src/client/smartAccountClient.ts#L122C38-L122C57

Reply:
oh I gotta use a different name, but this method is different from what sendRawUserOperation of bundler client does (just call rpc)

Reply:
it's just that the Bundler client's `sendRawUserOperation` requires account.getEntryPoint().address to be passed in as a param. So wanted to make the function formats consistent with other smart account client actions.

Reply:
hmmm good catch, but there's still the issue that the rest of bundler actions don't follow that same standard. We'd have to fix the inconsistency across the board. I'd rather keep the bundler actions and the smart account client actions consistent amongst themselves for now. We should fix ALL of the bundler client actions to match the object param style of the other clients, but we can't change that API right now

Reply:
I think we should not duplicate the actions here, and track that we need to refactor the bundler client action signatures next time we intro a breaking change (likely v4 or if need a breaking change in the entrypoint 0.7 support)